specs/centos_ci.adoc

```diff
@@ -0,0 +1,135 @@
+// vim: tw=79
+
+= CentOS CI
+
+We need to make sure that new code doesn't break existing codebase and check
+that new features works as early as possible. This can be done by running
+acceptance integration tests automatically, using CentOS CI infrastructure.
+
+== Problem description
+
+CentOS CI
+will be configured to launch acceptance integration test run for every new
+nightly rpm build from tendrl nightly copr project. As set of implemented
```

Comment:
@brainfunked yes copr has this capability. You can configure nightly/per pull request based builds